### ISSUE TITLE
Rails 3.1 & A.P.: Sprite creation fails because of missing Logger

### ISSUE THREAD

**Issue by ChristianPeters** [Compass/compass#567]
Environment: Rails 3.1.0, current chriseppstein/compass master branch, sass-rails 3.1.0; I created a blank rails app with asset pipeline enabled and specified a simple sprite in application.css.sass.

Stylesheet compilation fails on the following three lines with 'undefined method nil.[]':
- https://github.com/chriseppstein/compass/blob/master/lib/compass/sass_extensions/sprites/sprite_methods.rb#L63
- https://github.com/chriseppstein/compass/blob/master/lib/compass/sass_extensions/sprites/sprite_methods.rb#L69
- https://github.com/chriseppstein/compass/blob/master/lib/compass/sass_extensions/sprites/sprite_methods.rb#L101

If I comment out these lines or set `sass_options = {quiet: true}` in my `config/compass.rb`, everything is fine.

Setting `logger = STDOUT`in `config/compass.rb` did not help.

**Comment by unimatrixZxero** [comment]
+1

**Comment by nbarthelemy** [comment]
+1

**Comment by scottdavis** [comment]
damnit chris! /cc @chriseppstein !

**Comment by pzgz** [comment]
Thanks, this did help me out, just for correction, the option should be 'sass_options = {quite: true} ', that's a typo for 'quite'

**Comment by qnm** [comment]
Using :quiet => true didn't work for me. I used

```
sass_options = { :compass => { :logger => Compass::NullLogger.new } }
```

instead - ymmv.

**Comment by chriseppstein** [comment]
Fixed by fecc98219d05104a13f66be78bc2ca9d048ab369
